### ISSUE TITLE
Unable to load fonts

### ISSUE THREAD

**Issue by coopersamuel** [tamagui/tamagui#1580]
I've followed the expo guide to the letter, seems like things are working but the fonts are not loading. I'm seeing this error in the console. Upon inspection, there is no `src` directory included in the `@tamagui/font-inter` package

```
web compiled with 2 errors and 1 warning
WARNING in ./node_modules/@tamagui/font-inter/dist/esm/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/cooper/projects/ai-bible/bible-reader/node_modules/@tamagui/font-inter/src/index.ts' file: Error: ENOENT: no such file or directory, open '/Users/cooper/projects/ai-bible/bible-reader/node_modules/@tamagui/font-inter/src/index.ts'
```

**Comment by jpmmatias** [comment]
I have the same problem here, anyone found the solution?

**Comment by natew** [comment]
need a repro unfortunately, seems like its picking up source files when it should be picking up build files? even info on platform etc is useful

**Comment by coopersamuel** [comment]
I ended up scrapping my project and starting over with a new expo-tamagui-solito starter as mentioned in the docs. Since my project was in very early stages, this made the most sense for me and was the quickest way to get something working. Maybe @jpmmatias can provide some additional information to help reproduce? Overall I've encountered quite a few roadbumps while getting started with Tamagui for a basic use case with Expo and React Native on the latest versions
